pkg/kubelet/dockertools/manager.go

```diff
@@ -213,24 +216,25 @@ func NewDockerManager(
 	}
 
 	dm := &DockerManager{
-		client:                 client,
-		recorder:               recorder,
-		containerRefManager:    containerRefManager,
-		os:                     osInterface,
-		machineInfo:            machineInfo,
-		podInfraContainerImage: podInfraContainerImage,
-		dockerPuller:           newDockerPuller(client, qps, burst),
-		dockerRoot:             dockerRoot,
-		containerLogsDir:       containerLogsDir,
-		networkPlugin:          networkPlugin,
-		livenessManager:        livenessManager,
-		runtimeHelper:          runtimeHelper,
-		execHandler:            execHandler,
-		oomAdjuster:            oomAdjuster,
-		procFs:                 procFs,
-		cpuCFSQuota:            cpuCFSQuota,
-		enableCustomMetrics:    enableCustomMetrics,
-		configureHairpinMode:   hairpinMode,
+		client:                           client,
+		recorder:                         recorder,
+		containerRefManager:              containerRefManager,
+		os:                               osInterface,
+		machineInfo:                      machineInfo,
+		podInfraContainerImage:           podInfraContainerImage,
```

Comment:
Augmenting SGTM.

On Thu, Mar 31, 2016 at 2:08 PM, Andy Goldstein <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/23686#discussion_r58127940>
> :
>
> > -		containerLogsDir:       containerLogsDir,
> > -		networkPlugin:          networkPlugin,
> > -		livenessManager:        livenessManager,
> > -		runtimeHelper:          runtimeHelper,
> > -		execHandler:            execHandler,
> > -		oomAdjuster:            oomAdjuster,
> > -		procFs:                 procFs,
> > -		cpuCFSQuota:            cpuCFSQuota,
> > -		enableCustomMetrics:    enableCustomMetrics,
> > -		configureHairpinMode:   hairpinMode,
> > +		client:                           client,
> > +		recorder:                         recorder,
> > +		containerRefManager:              containerRefManager,
> > +		os:                               osInterface,
> > +		machineInfo:                      machineInfo,
> > +		podInfraContainerImage:           podInfraContainerImage,
>
> There's an ImageSpec in pkg/kubelet/container/runtime.go but it only has Image
> string in it. We could augment if desired?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23686/files/cbae73e072453c627e07b2a21eb0fc201ca5c984#r58127940>
>